db/connection_test.py

```diff
@@ -0,0 +1,10 @@
+from pymongo import MongoClient
+# pprint library is used to make the output look more pretty
+from random import randint
+# connect to MongoDB, change the << MONGODB URL >> to reflect your own connection string
```

Comment:
We can likely do this as a variable of some sort. Python supports templating as f-strings. Eg something like:

```python
username = "Gabe"
password = "Pa55w0rd!"
url = f'mongodb+srv://{username}:{password}@cluster0.27gwi.mongodb.net/Cluster0?retryWrites=True&w=majority'
client = MongoClient(url)

Reply:
Bit more flexible than a direct URL here essentially